aspects/aspects.go

```diff
@@ -1089,7 +1099,11 @@ func (s JSONSchema) Validate(jsonData []byte) error {
 	return json.Unmarshal(jsonData, &data)
 }
 
-// CheckTypeForPath always returns Any since it doesn't constrain values to specific types.
-func (v JSONSchema) TypeAtPath(path []string, index int) ([]SchemaType, error) {
-	return []SchemaType{Any}, nil
+// AtPath always the JSONSchema.
```

Comment:
I think this is missing some words? Also, a few other implementations of this interface are missing doc comments.